src/mahoji/lib/abstracted_commands/farmingContractCommand.ts

```diff
@@ -87,7 +88,9 @@ export async function farmingContractCommand(userID: string, input?: ContractOpt
 					)
 				).files,
 				components:
-					newContractLevel !== 'easy' ? makeComponents([makeEasierFarmingContractButton()]) : undefined
+					newContractLevel !== 'easy'
```

Comment:
Following my previous comment make all three of the "components:" like this:

newContractLevel !== 'easy'
						? makeComponents([makeEasierFarmingContractButton(), autoContractButton])
						: makeComponents([autoContractButton])

This also ensures that if the current contract level is easy the autocontractbutton is still shown

Reply:
To be honest I had the third one like that but then I got the The air that it could not find Something something or whatever So I had to put it back as input like it was

Reply:
You'll have to be more specific than "the last little piece", if you copy and paste in the code I gave you, remove the "import { makeAutoContractButton } from '../../../lib/util/globalInteractions';" and then lint you will be good to go

Reply:
if you can make edits you're more then welcome to. thank you for this feedback

Reply:
fixed it.